source/core_sntp_client.c

```diff
@@ -301,34 +300,29 @@ static SntpStatus_t sendSntpPacket( const UdpTransportInterface_t * pNetworkIntf
     getTimeFunc( &lastSendTime );
 
     /* Loop until the entire packet is sent. */
-    while( ( bytesRemaining > 0UL ) && ( sendError == false ) )
+    while( ( bytesSent != ( int32_t ) packetSize ) && ( sendError == false ) )
```

Comment:
It looks possible to do away with `sendError`, but it's just an extra byte of memory and IMO, easier to read.

Reply:
Yeah, I am not a fan of using `break`s 🙂